tests/mhd/test_spec.py

```diff
@@ -1,29 +1,35 @@
+from cmath import isnan
 import unittest
 import numpy as np
 import os
 import logging
 import shutil
 
 #from simsopt.mhd.spec import spec_found
-#if spec_found:
+# if spec_found:
 #    from simsopt.mhd.spec import Spec, Residue, pyoculus_found
 try:
     import spec
     spec_found = True
-except:
+except BaseException:
     spec_found = False
 
 try:
     import pyoculus
     pyoculus_found = True
-except:
+except BaseException:
     pyoculus_found = False
 
 try:
     from mpi4py import MPI
-except:
+except BaseException:
     MPI = None
 
+try:
+    from simsopt.mhd.profiles import SpecProfile
+except BaseException:
+    SpecProfile = None
+
```

Comment:
The try-except block seems unnecessary. Won't `from simsopt.mhd.profiles import SpecProfile` always succeed?